mcux/hal_nxp.cmake

```diff
@@ -115,8 +115,9 @@ include_driver_ifdef(CONFIG_COUNTER_MCUX_PIT		pit		driver_pit)
 include_driver_ifdef(CONFIG_COUNTER_MCUX_RTC		rtc		driver_rtc)
 include_driver_ifdef(CONFIG_DAC_MCUX_DAC		dac		driver_dac)
 include_driver_ifdef(CONFIG_DAC_MCUX_DAC32		dac32		driver_dac32)
-include_driver_ifdef(CONFIG_DMA_MCUX_EDMA		edma		driver_edma)
 include_driver_ifdef(CONFIG_DMA_MCUX_EDMA		dmamux		driver_dmamux)
+include_driver_ifdef(CONFIG_DMA_MCUX_EDMA_LEGACY	edma		driver_edma)
```

Comment:
There is some discussion on the Zephyr PR to not call this LEGACY. Kindly update this based on the new Kconfig name.

Reply:
Sure, i have updated